<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreTreat</class>
 <widget class="QMainWindow" name="PreTreat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PreTreat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="OpenImageButton">
          <property name="text">
           <string>打开图片</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="SaveImageButton">
          <property name="text">
           <string>保存图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="HighpassButton">
        <property name="text">
         <string>高通滤波（锐化）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ErosionButton">
        <property name="text">
         <string>腐蚀</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LowpassButton">
        <property name="text">
         <string>低通滤波（平滑）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SmoothButton">
        <property name="text">
         <string>平滑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>反色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SobelButton">
        <property name="text">
         <string>sobel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DilationButton">
        <property name="text">
         <string>膨胀</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LaplaceButton">
        <property name="text">
         <string>lapalce</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MedianpassButton">
        <property name="text">
         <string>中值滤波</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="thresholdButton">
        <property name="text">
         <string>threshold</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
